integration_tests/dbt_project.yml

```diff
@@ -20,6 +20,25 @@ vars:
   
   reddit_ads__ad_conversions_passthrough_metrics: 
     - name: 'view_through_conversion_attribution_window_week'
+  reddit_ads__account_passthrough_metrics: 
```

Comment:
Did we want to keep these new variable configurations for the default `dbt_project.yml` Buildkite runs, or should we comment them out?

Reply:
Yeah just wanted to make sure this passes on all warehouses. 

What do you mean by source file?

Reply:
Bleh, `dbt_reddit_ads_source`, just so the `dbt_project.yml` files are matching.

Something to do later though!

Reply:
Ah gotcha, yah later haha